lib/grpcx/server.rb

```diff
@@ -22,6 +21,29 @@ def handle(service)
 
         super(service)
       end
+
+      private
+
+      def apply_default_options(opts={})
```

Comment:
This doesn't need to be part of Methods, this can be a static method outside. No need to include (even private) methods into the grpc servers, if not necessary